.github/workflow/ci.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: build
+on: push
+jobs:
+  tox:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        tox:
+          - py310
+          - pre-commit
+          - mypy
+          - docs
```

Comment:
These three environments don't exist in tox.ini. You should probably just change this to `py38` and `py310`